UI/DOM.js

```diff
@@ -0,0 +1,115 @@
+class View {
+  constructor(model, isLogged) {
+    this.model = model;
+    this.isLogged = isLogged;
+  }
+
+  static createPostHtml(post, isLogged) {
+    const newPost = document.createElement('div');
+    newPost.innerHTML = `<div class = "post">
+        <div class="user_name">
+            ${post.author}
+        </div>
+        <div class="image">
+            <img src = ${post.photoLink} alt = "cat" width= "100%" >
+        </div>
+        <button type="submit" class="like"  >
+            <i class="far fa-heart fa-lg"  data-fa-transform="shrink-8 right-6"  data-fa-transform="shrink-8 down-116"></i>
+        </button>
+        ${isLogged ? `<button type="submit" class="edit">
+          <i class="far fa-edit"  data-fa-transform="shrink-8 right-6"  data-fa-transform="shrink-8 down-116"></i>
+          </button>
+          <button type="submit" class="delete"  >
+          <i class="fas fa-trash-alt"  data-fa-transform="shrink-8 right-6"  data-fa-transform="shrink-8 down-116"></i>
+          </button>` : ''} 
+        <time class="post-time" datetime=${post.create} title=${post.create}>
+        ${post.create}
+        </time>
+        <div class="text-container">
+            <p>
+                <b>${post.likes.length} likes </b>
+                <br><b>${post.author} </b> ${post.description}
+                <button  class="myBtn">show more</button>
+            </p>
+            <ul class="hashtags">
+            </ul>
+        </div>`;
+    const tagsBlock = newPost.querySelector('.hashtags');
+    newPost.dataset.id = post.id;
+    tagsBlock.innerHTML = post.hashtags.map(
+      tag => `<li class="tag"><a href="" >#${tag}</a></li>`
+    ).join('');
+    return newPost;
+  }
+
+  addPost(post) {
+    if (this.model.add(post)) {
+      const phototape = document.querySelector('.phototape');
```

Comment:
It's also will be nice to cache root element.
And it will be ideal to define it outside of class instance.